examples/srlinux/2node-srl-ixr6-with-oc-services.pbtxt

```diff
@@ -0,0 +1,114 @@
+# This topology consists of two Nokia SR Linux IXR-6e nodes running 22.6.3 release and connected with a single link over ethernet-1/1 interfaces.
+# Each of the nodes comes up with a startup config which is sourced from 22.6.3-opeconfig.cfg.json file;
+# the startup config contains a self-signed TLS cert and has gnmi, gnoi, gribi, p4rt services enabled
+#
+# Read more about this topology at http://learn.srlinux.dev/tutorials/infrastructure/kne/srl-with-kne/
+
+name: "2-srl-ixr6"
+nodes: {
+    name: "srl1"
+    vendor: NOKIA
+    model: "ixr6e"
+    config:{
+        image: "ghcr.io/nokia/srlinux:22.6.4"
+        file: "22.6.3-openconfig.cfg.json"
```

Comment:
should be updated to match

Reply:
I have renamed the file to `srl-openconfig.cfg.json` to make sure it can be used with future srl image versions without misguiding users as to which version it belongs to.